### ISSUE TITLE
Use custom icons as "tl-timemarker-content-container"

### ISSUE THREAD

**Issue by dasrecord** [NUKnightLab/TimelineJS3#844]
Deploying career timelines for each artist on an [music agency website.](https://evolvedmusicgroup.com)
GitHub for project available [here.](https://github.com/dasrecord/emg-website)

Main feedback from is that the vertical sizing/spacing of the timeline bar and event cards are too large in the X axis.
One of our artists asked if it was possible to replace the cards with a small icon instead.

Would love to contribute to this amazing tool.

**Comment by JoeGermuska** [comment]
In case you didn't realize, you can use the "media thumbnail" for a slide (column O in the spreadsheet, or in `.events[].media.thumbnail` in the JSON config -- in either case the value should be a URL that resolves to an image.
https://timeline.knightlab.com/docs/using-spreadsheets.html
https://timeline.knightlab.com/docs/json-format.html

This doesn't address the vertical sizing/spacing part. We readily admit that TimelineJS isn't well factored for custom CSS, since the logic and presentation are fairly entangled, and the CSS selectors tend to be overly specific, but see  https://timeline.knightlab.com/docs/overriding-styles.html for a little more on that.

We tend to be conservative about accepting changes to the codebase, so if the effort you'd spend changing the code is contingent on whether it's merged into this repo, we should talk about it before you do all the work -- but if making it work is more important than whether it gets merged, go for it!
